include/dlaf/memory/memory_view.h

```diff
@@ -55,7 +55,8 @@ class MemoryView {
   /// @param size The size (in number of elements of type @c T) of the existing allocation,
   /// @pre @p ptr+i can be deferenced for 0 < @c i < @p size.
```

Comment:
I know it is not related, but we might just take the chance to fix the doc.

```suggestion
  /// @pre @p ptr+i can be deferenced for 0 <= @c i < @p size.
```

Reply:
https://github.com/eth-cscs/DLA-Future/pull/941/commits/eb80189c7b6371686fdd8e6ea635cb59f7fee452. Also fixed `deferenced` to `dereferenced`, and the same problems in `memory_chunk.h`.